src/api/app/views/webui2/webui/package/show.html.haml

```diff
@@ -0,0 +1,71 @@
+:ruby
+  @pagetitle = "Show #{@project} / #{@package}"
+  @layouttype = 'custom'
+  devel_package = @package.develpackage
+  package_bread_crumb
+
+.card.mb-3
+  = render partial: 'tabs', locals: { project: @project, package: @package }
+  .card-body
+    .row
+      .col-md-8
+        %h3#package-title
+          = @package.title.presence || @package.name
+          - if @package.url.present?
+            %small= link_to(@package.url, @package.url)
+        #description-text
+          - if @package.description.blank?
+            %i No description set
+          - else
+            = simple_format(@package.description)
+      .col-md-4
+        = render partial: 'side_links', locals: { devel_package: devel_package,
+                                                  failures: @failures,
+                                                  linkinfo: @linkinfo,
+                                                  revision: @revision,
+                                                  requests: @requests,
+                                                  project: @project,
+                                                  package: @package }
+      = render partial: 'bottom_actions'
+.row
+  .col-lg-8
+    .card.mb-3
+      %h5.card-header
+        Source Files
+        = image_tag('ajax-loader.gif', id: 'spinner_files', style: 'display: none')
```

Comment:
fontawesome has [rotating icons](https://fontawesome.com/how-to-use/on-the-web/styling/rotating-icons)